<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addSiblings</class>
 <widget class="QDialog" name="addSiblings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Siblings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0" colspan="3">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="closeButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addSibling">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editSibling">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDateEdit" name="dateOfBirth">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="maximumDate">
      <date>
       <year>2100</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="minimumDate">
      <date>
       <year>1900</year>
       <month>9</month>
       <day>14</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Date of birth:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Type of Delivery</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="gender">
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Twins</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="typeOfDelivery">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Vaginal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C/S</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D&amp;C</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Complications</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="genericLineEdit" name="placeOfBirth"/>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Place of Birth</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Term</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="state">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>FullTerm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PreTerm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Abortion</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>IUFD</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="TextEdit" name="complications">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>textedit.h</header>
  </customwidget>
  <customwidget>
   <class>genericLineEdit</class>
   <extends>QLineEdit</extends>
   <header>genericlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
